hoomd/md/TwoStepNPTMTK.h

```diff
@@ -137,7 +137,7 @@ class PYBIND11_EXPORT TwoStepNPTMTK : public IntegrationMethodTwoStep
         virtual void integrateStepTwo(unsigned int timestep);
 
         //! Get needed pdata flags
-        /*! TwoStepNPTMTK needs the pressure, so the isotropic_virial or pressure_tensor flag is set,
+        /*! TwoStepNPTMTK needs the pressure, so the pressure_tensor flag is set,
             depending on the integration mode
```

Comment:
Does this depend on the integration mode anymore?
```suggestion
        /*! TwoStepNPTMTK needs the pressure, so the pressure_tensor flag is set.
```